crates/biome_js_analyze/src/lint/complexity/use_literal_keys.rs

```diff
@@ -74,6 +74,17 @@ impl Rule for UseLiteralKeys {
         };
         let value = inner_expression.as_static_value()?;
         let value = value.as_string_constant()?;
+        // Bail if the value contains unescaped new line characters
+        // that can only appear in template strings.
+        //
+        // const a = {
+        //   `line1
+        //   line2`: true
+        // }
+        //
+        if has_unescaped_new_line(value) {
```

Comment:
Do you think we could add this to the following if (line 97):

```js
if matches!(node, AnyJsMember::JsComputedMemberName(_)) || is_js_ident(value) {
```

as follows:

```js
if (matches!(node, AnyJsMember::JsComputedMemberName(_)) && has_unescaped_new_line(value)) || is_js_ident(value) {
```

`is_js_ident` should rule out newlines.

Reply:
Yes, this further clarifies the logic.